subt_ign/src/GameLogicPlugin.cc

```diff
@@ -696,6 +713,25 @@ void GameLogicPlugin::PostUpdate(
   auto currentTime = std::chrono::steady_clock::now();
   if (currentTime - this->dataPtr->lastStatusPubTime > std::chrono::seconds(1))
   {
+    ignition::msgs::Clock competitionClockMsg;
+    ignition::msgs::Header::Map *mapData =
+      competitionClockMsg.mutable_header()->add_data();
+    mapData->set_key("phase");
+    if (this->dataPtr->started)
+    {
+      mapData->add_value(this->dataPtr->finished ? "finished" : "run");
+      auto secondsRemaining = std::chrono::duration_cast<std::chrono::seconds>(
+          remainingCompetitionTime);
+      competitionClockMsg.mutable_sim()->set_sec(secondsRemaining.count());
+    }
+    else if (!this->dataPtr->finished)
```

Comment:
Do we have a case where `started = false` and `finished = true`?

Reply:
Looks like a team could trigger finish before starting. Handled that case in 607c6de91d9baa5483bb824e18ba42aae172b934